### ISSUE TITLE
About publishing software using VailDEC3 source

### ISSUE THREAD

**Issue by rinasvideo** [Vail-Zero/VailDEC3-sources#5]
Can I publish software created using your VailDEC3 source code?

**Comment by Vail-Zero** [comment]
Yes, you are free to redistribute it if you follow the wiki and license.

**Comment by rinasvideo** [comment]
Should I distribute the license display with the executable file?

**Comment by Vail-Zero** [comment]
For now, all you have to do is display it on the download site, if you enclose it in the next build.

**Comment by rinasvideo** [comment]
thank you
